### DISCUSSION TITLE
Exporting to DOCX References

### DISCUSSION THREAD

**Discussion by vanessarosa710** [knowledgefutures/pubpub#2661] [Support]
One of the most valuable aspects of using PubPub is its capacity to use rich media, reference research, and export to various formats. However, exports to Word (arguably the most used in academia) are not functional and result in an irrational string of letters without reference to the original citation. Please advise or correct this issue as it substantively reduces the utility of PubPub. Thank you.

![Screenshot 2023-05-09 at 8 24 11 AM](https://github.com/pubpub/pubpub/assets/132926293/a0fba696-67e9-462c-8bf0-da50a10d8c7f)
![Screenshot 2023-05-09 at 8 25 14 AM](https://github.com/pubpub/pubpub/assets/132926293/70479a89-61c0-4e9c-8193-8e91f3272c99)

**Comment by gabestein** [comment]
Thanks for letting us know. We're aware and will be taking a look. It does look like this currently applies to rich text citations only, not ones generated via DOI, bibtex, etc., so a workaround for now is using generated citations.
